core/src/main/groovy/noe/common/utils/Cmd.groovy

```diff
@@ -821,12 +821,10 @@ public class Cmd {
      */
     Process psProc = psCommand.execute()
     Process proc2 = ['grep', identifier].execute()
-    // This is a big assumption, but I guess both EAP and Catalina have .sh start scripts...
-    Process proc3 = ['grep', '.sh'].execute()
```

Comment:
@sgala This change must also be checked by @jstourac or @honza-kasik as it may impact them

Reply:
There is no ".sh" in the process output. The code was `ps -ef | grep 1471948789 | grep .sh | grep -v grep` except for HP-UX where it was `ps -elfx | grep 1471948789 | grep .sh | grep -v grep`. I don't know for HP-UX but for linux and current JWS versions there is no .sh in the ps line and the pipeline was always empty. It works as it is now.

The comment already says the next line is A BIG ASSUMPTION. The .sh in catalina start script is gone for the running process. You can check it.

Reply:
Please remember that this project used by others too, and need to made aware of changes that might impact them.

Reply:
Thanks @PaulLodge . Exactly as you say. EAP uses set of scripts in `$EAP_HOME/bin` to start. Specifically `standalone.sh` and `domain.sh`.